### PR TITLE
Be able to dynamically configure raster base path

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

File: doc/integrator/objectstorage.rst [hunk 1]
```diff
@@ -177,6 +177,13 @@ Azure:
 
 `Some more information <https://github.com/mapserver/mapserver/wiki/Render-images-straight-out-of-S3-with-the-vsicurl-driver>`_
 
+.. note::
+
+   If you want to be able to use different Bucket or Container in different environment
```
Comment (jwkaltz) [inline doc/integrator/objectstorage.rst:182]: Suggestion:

If you want to use different buckets or containers in different environments (such as integration / production)

File: doc/integrator/objectstorage.rst [hunk 2]
```diff
@@ -177,6 +177,13 @@ Azure:
...
+   (like integration / production), you should have a file names ``<name>.raster``
```
Comment (jwkaltz) [inline doc/integrator/objectstorage.rst:183]: I don't understand what these file names refer to.

Maybe an example would help.

Comment (sbrunner) [reply doc/integrator/objectstorage.rst]: I try to clarify ...

File: doc/integrator/objectstorage.rst [hunk 3]
```diff
@@ -177,9 +177,20 @@ Azure:
 
 `Some more information <https://github.com/mapserver/mapserver/wiki/Render-images-straight-out-of-S3-with-the-vsicurl-driver>`_
 
+.. note::
 
-QGIS client
------------
+   If you want to use different buckets or containers in different environments
+   (such as integration / production), you should add an empty file names ``<name>.raster``
```
Comment (arnaud-morvan) [inline doc/integrator/objectstorage.rst:183]:
```suggestion
   (such as integration / production), you should add an empty file named ``<tileindexbasename>.raster``
```

File: doc/integrator/objectstorage.rst [hunk 4]
```diff
@@ -177,9 +177,20 @@ Azure:
...
+   (not ``<name>.shp.raster``) and a ``RASTER_BASE_PATH`` environment variable in your
+   config container, then the base path will be replaced (same number of folder).
```
Comment (arnaud-morvan) [inline doc/integrator/objectstorage.rst:185]:
```suggestion
   config container, then the base path will be replaced (same number of folders).
```

**Comment by sbrunner** [comment]
Change the option to don't fail on the build of the config

File: doc/integrator/objectstorage.rst [hunk 5]
```diff
@@ -209,3 +222,30 @@ Then you can add a raster layer with:
...
+
+* ``AZURE_STORAGE_CONNECTION_STRING``: The connection string.
+
+For Azure AKS the access should be given by the AzureAssignedIdentity in Kubernetes,
+
+.. note::
+
+   If you want to use different buckets or containers in different environments
+   (such as integration / production), you should add an empty file names
+   ``<name>.ggs.raster`` or ``<name>.ggz.raster``
```
Comment (arnaud-morvan) [inline doc/integrator/objectstorage.rst:248]:
```suggestion
   ``<name>.qgs.raster`` or ``<name>.qgz.raster``
```

**Review by arnaud-morvan — APPROVED** [review]
LGTM, maybe desserve some tests on eval-templates script.

**Review by arnaud-morvan — COMMENTED** [review]
eval-templates also desserve a docstring / description about conversions / tasks it executes.

**Comment by sbrunner** [comment]
> eval-templates also desserve a docstring / description about conversions / tasks it executes.

Done in the command help :-)

**Pull request merged**